events/views.py

```diff
@@ -1,11 +1,48 @@
 from django.shortcuts import render
 from django.http import HttpResponse
+from django.shortcuts import render, redirect
+from django.views.generic.edit import CreateView
 
-from .forms import EventPlaceForm, EventCategoryForm
+
+from .forms import EventForm, EventFormSet, EventPlaceForm, EventCategoryForm
+
+
+class EventCreateView(CreateView):
+    form_class = EventForm
+    template_name = "events/create_event.html"
+
+    def get_context_data(self, **kwargs):
+        context = super(EventCreateView, self).get_context_data(**kwargs)
+        context["event_formset"] = EventFormSet()
+        return context
+
+    def post(self, request, *args, **kwargs):
+        self.object = None
+        form = self.get_form()
```

Comment:
Use CreateWithInlinesView of extra views. Dont override post method. Changing self.object to None looks fishy